### ISSUE TITLE
[REQUEST]

### ISSUE THREAD

**Issue by PokeBotUK** [PocketMobsters/MobBotTemp#90]
On Discord:

_"Going in a straight line between stops. This was fixed, sort of, with our human pathing system but the server we point them at is public and used by other services and consistently is overloaded so we have it default back to straight lines when it fails. This is fixed as of v2.0.2 because we have our own routing server now."_

1) Any chance this could be configurable ? ie run our own pathing server and if so
2) What software needs to be installed - Open Source Routing Machine ? (http://wiki.openstreetmap.org/wiki/Open_Source_Routing_Machine)

Just my 2c

**Comment by jjskuld** [comment]
It's a good idea.. I just submitted a patch to fix the human pathing back to the public servers because their "own routing server" was returning errors.  Sounds like making this configurable would solve this issue so you could pick a server that was up and running if the other was down. I can submit a new pull request but they haven't even accepted my first pull #92 yet, so waiting on that first.
